package.json

```diff
@@ -98,15 +98,15 @@
     "eslint-plugin-react-hooks": "^4.2.0",
     "jest": "^27.0.4",
     "microbundle-crl": "^0.13.11",
-    "postcss": "^8.3.6",
+    "postcss": "^7.0.38",
```

Comment:
I kept receiving this `Error: PostCSS plugin tailwindcss requires PostCSS 8.` error on my local when attempting to start up Storybook so I followed TW's documentation to downgrade 🤔 
https://tailwindcss.com/docs/installation#post-css-7-compatibility-build